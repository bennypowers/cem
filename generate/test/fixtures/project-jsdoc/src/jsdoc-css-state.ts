/**
 * @cssstate state
 * @cssstate state-description - state description
 * @cssstate state-description-multiline - multiline
 *                                         state description
 */
@customElement('jsdoc-css-state')
class JsdocCssState extends LitElement { }
