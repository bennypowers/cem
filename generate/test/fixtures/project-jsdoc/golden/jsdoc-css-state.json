{
  "schemaVersion": "2.1.0",
  "modules": [
    {
      "kind": "javascript-module",
      "path": "src/jsdoc-css-state.js",
      "declarations": [
        {
          "name": "JsdocCssState",
          "superclass": {
            "name": "LitElement",
            "package": "lit"
          },
          "source": {
            "href": "https://github.com/bennypowers/cem/tree/main/generate/test/fixtures/project-yaml/src/jsdoc-css-state.ts#L7"
          },
          "kind": "class",
          "tagName": "jsdoc-css-state",
          "cssStates": [
            {
              "name": "state"
            },
            {
              "name": "state-description",
              "description": "state description"
            },
            {
              "name": "state-description-multiline",
              "description": "multiline\nstate description"
            }
          ],
          "customElement": true
        }
      ],
      "exports": [
        {
          "kind": "custom-element-definition",
          "name": "jsdoc-css-state",
          "declaration": {
            "name": "JsdocCssState",
            "module": "src/jsdoc-css-state.js"
          }
        }
      ]
    }
  ]
}